public/content/sv/welcome/install-d-locally.md

```diff
@@ -0,0 +1,43 @@
+# Installera D lokalt
+
+På D språkets webbsida [dlang.org](https://dlang.org) kan den senaste kompilerar
+versionen av referens kompilatorn **DMD** (Digital Mars D)
+[laddas ned](http://dlang.org/download.html) och installeras:
+
+### Windows
+
+* [Installerare](http://downloads.dlang.org/releases/2.x/{{latest-release}}/dmd-{{latest-release}}.exe)
+* eller: [Arkiv](http://downloads.dlang.org/releases/2.x/{{latest-release}}/dmd.{{latest-release}}.windows.7z)
+* genom [chocolatey](https://chocolatey.org/packages/dmd): `choco install dmd`
+
+### Mac OS X
```

Comment:
It should be `OS X` or `macOS` (this is the latest name for the latest version which has not been released yet). It has not been called `Mac OS X` for several releases (not specific for Swedish).